dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -130,7 +130,7 @@ USER root
 # Install required dependencies from wolfi repository
 RUN for iter in {1..10}; do \
         apk update && \
-        apk add --no-interactive --no-progress --no-cache nodejs-18=18.20.6-r0 npm=11.1.0-r0 glib dbus-glib libatk-1.0 \
+        apk add --no-interactive --no-progress --no-cache nodejs-18=18.20.7-r0 npm=10.9.2-r0 glib dbus-glib libatk-1.0 \
```

Comment:
I'm just puzzled because `npm` is downgraged here... Is this intended?

Reply:
@belimawr Yes, upgrade to npm 11 was performed without checking the required compatibility and was never caught on wolfi since it is not installing with engine validation. This change brings parity with the non-wolfi version and it is a strictly supported version for the corresponding node engine.

Reply:
Thanks for clarifying. Could you also mention this in the final commit message? It is a bit confusing to only have one of two changes mentioned in the PR/Commit.